File: app/src/test/java/com/navin/melalwallet/utils/UtilityTest.java
package com.navin.melalwallet.utils;

import org.junit.Test;

import static org.junit.Assert.*;

public class UtilityTest {

    Utility utility = new Utility();

    @Test
    public void getMin() {




        assertEquals(utility.getMin(3,3),3);


    }

    @Test
    public void getZoj() {

        assertTrue(utility.getZoj(2));

    }
}

File: app/src/main/java/com/navin/melalwallet/utils/UtilityTest.kt
package com.navin.melalwallet.utils

class UtilityTest {


    fun getZoj(i: Int): Boolean {
        return if (i % 2 == 0) {
            true
        } else {
            false
        }
    }

    companion object {


        fun getMin(a: Int, b: Int): Int {

            return if (a > b) {
                b
            } else {
                a
            }
        }
    }

}
